tests/behavior/transforms/remap.toml

```diff
@@ -215,14 +215,14 @@
       "b.equals" = "10"
       "c.equals" = "10"
       "d.equals" = "10"
-      "e.equals" = "20"
-      "f.equals" = "20"
+      "e.equals" = ""
+      "f.equals" = ""
```

Comment:
Because `to_string(null)` now returns `""`, these tests had to be updated.

However, I'm pretty sure that change was made in earlier PRs that are in master now, so I'm unsure why the CI didn't fail?